packages/diagram_capture/lib/diagram_capture.dart

```diff
@@ -2,6 +2,10 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+// TODO(TahaTesser): Update this package for deprecated SingletonFlutterWindow API
+// https://github.com/flutter/flutter/issues/127585 and remove this ignore.
+// ignore_for_file: deprecated_member_use
```

Comment:
would be better to just ignore the lines where this is used. Otherwise this will hide other deprecations in the future as well.

Reply:
Thank you! That makes sense